packages/ethereum-contracts/ops-scripts/libs/common.js

```diff
@@ -66,6 +67,40 @@ function detectTruffle() {
     return truffleDetected;
 }
 
+// extracts the gas related config for the given network from the truffle config
+// returns an object with the relevant fields set in the config (empty if none)
+//
+// NOTE: This implememtation only works for settings provided by a network specific config,
+// not for settings provided by a wildcard (network_id: "*") config.
+function getGasConfig(networkId) {
```

Comment:
:hot_face:

Reply:
yeah, I hate it too. But deep diving into a deprecated framework in order to figure out something less hacky doesn't seem like a good investment of time.